### ISSUE TITLE
How do we decide who can use the library?

### ISSUE THREAD

**Issue by ZooeyMiller** [dwyl/library#6]
We will be using github OAuth for this app, so we should be able to get information about who is a part of dwyl and who isn't, do we want to limit the use of the library to dwylers? Or should anyone with a GH account be allowed to sign up and use the library?

**Comment by ZooeyMiller** [comment]
Pros of anyone using the library:
+ Sharing knowledge
+ More input from external people on what books we should have

Cons of anyone using the library:
+ Cannot guarantee responsible use of the application if random people are able to use it. 
+ Hard to follow up on missing books if the users are not known/responsible
+ Would require some kind of authentication that someone has taken a book. If anyone who can sign in to the app can mark a book as taken out, then we cannot know whether they have actually taken that book out physically. With a smaller user group that we have contact with (dwyllers or otherwise) we could follow up on withdrawals.

**Comment by iteles** [comment]
We don't want to go down the route of user management just yet, this is a world onto itself and we'll deal with that as a separate module.

For now, I suggest that we identify whether the user is a member of the dwyl organisation or not, but leave the application open to everyone.

Eventually, we'll have keycard which will double for door entry and things like the library.
